pull-request/src/person.js

```diff
@@ -12,4 +12,5 @@ Person.prototype.fullname = function() {
 
 Person.prototype.wishHappyBirthday = function() {
     return "Happy birthday " + this.firstName;
+    this.age++;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this code after the "return" statement. ([javascript:UnreachableCode](http://localhost:9075/coding_rules#rule_key=javascript%3AUnreachableCode))

[See it in SonarQube](http://localhost:9075/project/issues?id=training%3Apull-request&pullRequest=45&issues=AWgn1ZDVJoAEVqwe_2LY&open=AWgn1ZDVJoAEVqwe_2LY)

[ISSUE_KEY]: AWgn1ZDVJoAEVqwe_2LY